CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ download_project(PROJ                googletest
 # when building with Visual Studio
 set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
 
-add_subdirectory(${googletest_SOURCE_DIR} ${googletest_BINARY_DIR})
+add_subdirectory(${googletest_SOURCE_DIR} ${googletest_BINARY_DIR} EXCLUDE_FROM_ALL)
```

Comment:
Unfortunately, the EXCLUDE_FROM_ALL option doesn't always work. In particular, until the recent CMake 3.7.0 release, it breaks the build when using the Xcode generator. Details of the issue can be found here:

https://gitlab.kitware.com/cmake/cmake/issues/16101

If the feature was reliable, I'd be happy to accept this change, but since this example needs to work for people using CMake versions prior to 3.7.0 and I want to keep the example as simple as possible, I can't really accept it.